mne/minimum_norm/tests/test_inverse.py

```diff
@@ -891,4 +898,37 @@ def test_inverse_ctf_comp():
         apply_inverse_raw(raw, inv, 1. / 9.)
 
 
+def _check_delayed_data(inst, delayed):
+    """Check whether data is represented as kernel or not."""
+    if delayed:
+        assert isinstance(inst._kernel, np.ndarray)
+        assert isinstance(inst._sens_data, np.ndarray)
+        assert inst._data is None
+        assert not inst._kernel_removed
+    else:
+        assert inst._kernel is None
+        assert inst._sens_data is None
+        assert isinstance(inst._data, np.ndarray)
+
+
+@testing.requires_testing_data
+@pytest.mark.parametrize('pick_ori', ['normal', 'vector'])
+def test_delayed_data(epochs, pick_ori):
+    """Test if kernel in apply_inverse_epochs was properly applied."""
+    inverse_operator = read_inverse_operator(fname_full)
+    inverse_operator = prepare_inverse_operator(inverse_operator, nave=1,
+                                                lambda2=lambda2,
+                                                method="dSPM")
```

Comment:
instead of reusing the variable I would write it like this:
```py
    inv = prepare_inverse_operator(orig=read_inverse_operator(fname_full),
                                   nave=1, lambda2=lambda2, method='dSPM')
```